asreview/data/statistics.py

```diff
@@ -0,0 +1,203 @@
+# Copyright 2019-2020 The ASReview Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import numpy as np
+
+
+def n_records(data):
+    """Return the number of records.
+
+    Arguments
+    ---------
+    data: asreview.data.ASReviewData
+        An ASReviewData object with the records to review.
+
+    Return
+    ------
+    int:
+        The statistic
+    """
+    return len(data)
+
+
+def n_included(data):
+    """Return the number of included records.
+
+    Arguments
+    ---------
+    data: asreview.data.ASReviewData
+        An ASReviewData object with the records to review.
```

Comment:
with the unlabeled records 

because not all records will be reviewed in ASReview... that is, you want to reduce the number of records to be reviewed to a minumum